packages/progress-bar/src/vaadin-progress-bar-styles.js

```diff
@@ -29,4 +29,15 @@ export const progressBarStyles = css`
   :host([dir='rtl']) [part='value'] {
     transform-origin: 100% 50%;
   }
+
+  @media (forced-colors: active) {
+    [part='bar'] {
+      outline: 1px solid;
+    }
+
+    [part='value'] {
+      background-color: AccentColor !important;
```

Comment:
Any other places where we should use `AccentColor`, like a checked Checkbox?

Reply:
I don't know – the thing is that forced-colors emulation in Chrome does not seem to use any colors except black, and I have not had the opportunity to test with Parallels or some other Windows VM yet, so I don't know exactly how those look in WHCM that actually uses those colors, and it's unclear whether various browsers even support them. There is also precious little info on how those system colors should be used.

So I decided to only use it here where I needed to specify _some_ specific color, and I can't force it to black since that would not work well in a dark-mode forced-colors mode (which I think is the default for WHCM).